### ISSUE TITLE
Move Away From Scala App

### ISSUE THREAD

**Issue by szareiangm** [snowplow/snowplow-elasticsearch-loader#105]
As was Suggested in a pull request.

@BenFradet what is your suggestion? Should we follow some similar pattern like stream-enricher?

**Comment by BenFradet** [comment]
Yup let's just have a good old `def main(args: Array[String]): Unit`
